src/test/java/plugins/AnalysisCollectorPluginTest.java

```diff
@@ -59,6 +61,7 @@
  * @author Ullrich Hafner
  */
 @WithPlugins({"analysis-collector", "checkstyle", "pmd", "findbugs", "tasks", "warnings"})
+@Ignore("replaced by Warnings Next Generation Plugin (warnings-ng)")
```

Comment:
Can we delete these right away?

Reply:
Yes, of course. I just was not sure about that. I just disabled them to speedup the build.

Reply:
Should I remove them in this PR or in a new one?

Reply:
The sooner the better if we do not intend to fix them.

Reply:
A fix would be trivial, all publishers are now called `[Deprecated] ...`. But I do not need them anymore so I think it is ok to delete.